experiment/reporter.py

```diff
@@ -42,13 +42,19 @@ def output_report(experiment_config: dict,
                   coverage_report=False):
     """Generate the HTML report and write it to |web_bucket|."""
     experiment_name = experiment_utils.get_experiment_name()
-    web_filestore_path = posixpath.join(experiment_config['report_filestore'],
-                                        experiment_name)
-
     reports_dir = get_reports_dir()
 
-    core_fuzzers = yaml_utils.read(CORE_FUZZERS_YAML)['fuzzers']
-    fuzzers = sorted(set(experiment_config['fuzzers']).union(set(core_fuzzers)))
+    core_fuzzers = set(yaml_utils.read(CORE_FUZZERS_YAML)['fuzzers'])
+    experiment_fuzzers = set(experiment_config['fuzzers'])
+    fuzzers = sorted(experiment_fuzzers.union(core_fuzzers))
+
+    # Calculate path to store report files in filestore.
+    web_filestore_path = experiment_config['report_filestore']
+    if experiment_fuzzers != core_fuzzers:
+        # This means that we are running an experimental report with fuzzer
+        # variants. So, store these in |experimental| sub-directory.
```

Comment:
nit: is experimental == variants? i think we might consider some new engines experimental too that are not variants. how about "experiments running other than the default set of fuzzers with their default configuration" or something like that?

Reply:
experimental == variants + fuzzers not in core fuzzers. maybe that is not right. since like manul, eclipser are right now removed from core-fuzzers.yaml. so if we run an experiment with them, it will be considered experimental. adding another list might be needed. lets discuss with Jonathan on Tuesday, but feel free to put your thoughts here.

Reply:
i think one list is fine. (that's another question what should be in the list and what not.) my comment referred to changing the wording "experimental report with fuzzer variants" to something like "report with fuzzers not in the core list" to make it consistent with the code